.github/workflows/jenkins-dispatch.yml

```diff
@@ -12,35 +12,35 @@ on:
 
 jobs:
 
-  setup:
+  helm-tests:
     runs-on: ubuntu-latest
     steps:
-      - uses: percona-platform/checkout@v3
+
+      - name: checkout code
+        uses: percona-platform/checkout@v3
         with:
           submodules: 'recursive'
-      - uses: percona-platform/setup-node@v3
+
+      - name: install npm
+        uses: percona-platform/setup-node@v3
         with:
           node-version: '14'
-      - run: npm install -g bats
 
-  minikube_start:
-    needs: setup
-    runs-on: ubuntu-latest
-    steps:
-      - run: |
-          ls -la
-          ls -la ./sources/pmm-qa/src/github.com/percona/pmm-qa/pmm-tests/
-          ./sources/pmm-qa/src/github.com/percona/pmm-qa/pmm-tests/install_k8s_tools.sh --minikube
+      - name: install tools
+        run: |
+          npm install -g bats
+          ./sources/pmm-qa/src/github.com/percona/pmm-qa/pmm-tests/install_k8s_tools.sh --minikube --helm --kubectl --user
```

Comment:
we could just use a variable to set the path to pmm-qa and use it everywhere we invoke it

Reply:
ok, will do in a next iteration. To report back status of commit I need more inputs. So new PRs would follow.